corehq/apps/userreports/tests/test_report_builder.py

```diff
@@ -344,6 +344,32 @@ def test_reference_for_cases(self):
         assert 'first_name' == first_name_prop.get_id()
         assert 'first_name' == first_name_prop.get_text()
 
+    @flag_enabled('SHOW_IDS_IN_REPORT_BUILDER')
+    def test_show_case_id(self):
+        case_data_source = get_case_data_source(self.app, self.case_type)
+        case_data_source.save()
+        builder = ApplicationCaseDataSourceHelper(
+            self.domain,
+            self.app,
+            DATA_SOURCE_TYPE_CASE,
+            self.case_type,
+        )
+        expected_property_names = [
+            'closed',
+            'closed_on',
+            'first_name',
+            'last_name',
+            'modified_on',
+            'name',
+            'opened_on',
+            'owner_id',
+            'user_id',
+            'computed/owner_name',
+            'computed/user_name',
+            'case_id',
```

Comment:
nit: should this care about all columns and not just case_id? In that case how about renaming the test to `test_all_expected_property_names_when_flag_enabled`.
Or else limit this to testing just the case_id being present.
I see there is a test for other property names above this, so looks like other property names have already been tested for.